src/components/menu/menu-item/menu-item.component.js

```diff
@@ -115,19 +117,25 @@ const MenuItem = ({
         role="presentation"
         menuType={menuContext.menuType}
         display="inline-block"
+        m={rest.m}
```

Comment:
This breaks `styled-system/space`. Because `m=undefined` takes precedence over `ml`/`mr` etc.

We're spreading `rest` into this component so we can simple remove these two lines.

Reply:
This was my mistake, I was setting `mr` on an element which had nothing after it.

Reply:
Although these props are already included in `...rest` so I don't think we need to explicitly include them.